tests/test_integration.py

```diff
@@ -1806,10 +1765,12 @@ def explode_pex(path):
                 zf.extractall(path=destination_dir)
                 return [os.path.join(destination_dir, member) for member in sorted(zf.namelist())]
 
-        pexes = create_multiple_pexes()
+        pexes = run_command_with_jitter(
+            path_argument="--output-file", args=create_pex_command(quiet=True), count=3
+        )
         pex_members = {pex: explode_pex(path=pex) for pex in pexes}
-
-        for pex1, pex2 in itertools.combinations(pexes, r=2):
+        pex1 = pexes.pop()
+        for pex2 in pexes:
```

Comment:
N.B.: The LHS was always doing too much work. For `[0, 1, 2, ... N]`, to prove each pairing is equal you just need to compare `(0, 1), (0, 2) ... (0, N)` - transitivity handles the rest. Its only if you want to show that each element is unique that you need all combinations to compare different (used elsewhere in this PR).